src/Microsoft.TestPlatform.CoreUtilities/FeatureFlag/FeatureFlag.cs

```diff
@@ -47,6 +47,20 @@ internal partial class FeatureFlag : IFeatureFlag
 
     // Faster JSON serialization relies on less internals of NewtonsoftJson, and on some additional caching.
     public const string DISABLE_FASTER_JSON_SERIALIZATION = VSTEST_ + nameof(DISABLE_FASTER_JSON_SERIALIZATION);
+
+    public const string DISABLE_MULTI_TFM_RUN = VSTEST_ + nameof(DISABLE_MULTI_TFM_RUN);
```

Comment:
flag name here looks self explicative today, but maybe a pair of line of explanation could be good.

Reply:
added